src/dbus_api/api.rs

```diff
@@ -64,33 +63,23 @@ fn create_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
         Ok(pool_uuid) => {
             let pool_object_path: dbus::Path =
                 create_dbus_pool(dbus_context, object_path.clone(), pool_uuid);
-            let default_return =
-                MessageItem::Struct(vec![MessageItem::ObjectPath(default_object_path()),
-                                         MessageItem::Array(vec![], "o".into())]);
+            let default_return: Vec<(dbus::Path, Vec<dbus::Path>)> = Vec::new();
```

Comment:
I bet that you can get rid of the ```default_object_path``` method entirely with this commit, regardless, so that would be a good thing to remember to do.